upgrade_guide/migrating_from_astyanax/configuration/README.md

```diff
@@ -0,0 +1,242 @@
+
+### Timeouts
+
+Timeouts concerning requests, or connections will be part of the `SocketOptions`.
+
+*Astyanax*:
+
+```java
+ConnectionPoolConfigurationImpl cpool =
+       new ConnectionPoolConfigurationImpl("myConnectionPool")
+               .setSocketTimeout(3000)
+               .setConnectTimeout(3000)
+```
+
+*Java Driver :*
+
+```java
+SocketOptions so =
+       new SocketOptions()
+           .setReadTimeoutMillis(3000)
+           .setConnectTimeoutMillis(3000);
+```
+
+## Load Balancing
+Both *Astyanax* and the *Java driver* connect to multiple nodes of the *Cassandra*
+cluster. Distributing requests through all the nodes plays an important role in 
+the good operation of the `Cluster` and for best performances. With *Astyanax*, 
+requests (or “operations”) can be sent directly to replicas that have a copy of 
+the data targeted by the *“Row key”* specified in the operation. Since the *Thrift* API is
+low-level, it forces the user to provide *Row keys*, known as the `TokenAware` 
+connection pool type. This setting is also present in the *Java driver*, however 
+the configuration is different and provides more options to tweak.
+
+Load balancing in the *Java driver* is a *Policy*, it is a class that will be
+plugged in the *Java driver*’s code and the Driver will call its methods when it 
+needs to. The *Java driver* comes with a preset of specific load balancing policies. 
+Here’s an equivalent code :
+
+*Astyanax*:
+
+```java
+final ConnectionPoolType poolType = ConnectionPoolType.TOKEN_AWARE;
+final NodeDiscoveryType discType = NodeDiscoveryType.RING_DESCRIBE;
+ConnectionPoolConfigurationImpl cpool =
+       new ConnectionPoolConfigurationImpl("myConnectionPool")
+               .setLocalDatacenter("myDC")
+AstyanaxConfigurationImpl aconf =
+       new AstyanaxConfigurationImpl()
+               .setConnectionPoolType(poolType)
+               .setDiscoveryType(discType)
+```
+
+*Java driver*:
+
+```java
+LoadBalancingPolicy lbp = new TokenAwarePolicy(
+       DCAwareRoundRobinPolicy.builder()
+       .withLocalDc("myDC")
+       .build()
+);
+```
+
+*By default* the *Java driver* will instantiate the exact Load balancing policy 
+shown above, with the `LocalDC` being the DC of the first host the driver connects 
+to. So to get the same behaviour than the *TokenAware* pool type of *Astyanax*, 
+users shouldn’t need to specify a load balancing policy since the default one 
+should cover it.
+
+Important : Note that since *CQL* is an abstraction of the Cassandra’s architecture, a simple 
+query needs to have the *Row key* specified explicitly on a `Statement` in order 
+to benefit from the *TokenAware* routing (the *Row key* in the *Java driver* is 
+referenced as *Routing Key*), unlike the *Astyanax* driver. 
+Some differences occur related to the different kinds of `Statements` the *Java
+driver* provides. Please see [this link](../../../manual/statements) for specific information.
```

Comment:
It would be more appropriate to link to `../../../manual/load_balancing/#token-aware-policy`, it details how to set the routing key for each statement type.